src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WebSearch/Pages/WebSearchListPage.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Microsoft.CmdPal.Ext.WebSearch.Commands;
+using Microsoft.CmdPal.Ext.WebSearch.Helpers;
+using Microsoft.CmdPal.Ext.WebSearch.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+using BrowserInfo = Wox.Plugin.Common.DefaultBrowserInfo;
```

Comment:
I was looking to see if we could just do something else to run the default browser... but apparently our own docs just say to use Process.Start... 😢 https://learn.microsoft.com/troubleshoot/developer/visualstudio/csharp/language-compilers/start-internet-browser

Though I wonder if we could do something with Launcher.LaunchUriAsync, but then we'd need a setting for the base url for search (which could be cool?)

Reply:
I was so disappointed when I realized that it was only going to be the default search _engine_ and not the default browser. It would be really way better if we can eventually just make it the default browser. 

I think we'll probably get feedback from other folks who want the same.

Reply:
@zadjii-msft - we could in theory just take out the code from existing Wox. I've done that on a few occasions already... this one was just so messy and I figured we may as well not do that for now. 

PowerToys Run literally just has the Wox projects inside the launcher project, we could theoretically do the same?